packages/dotnet-sdk/README.md

```diff
@@ -391,20 +418,40 @@ private void printLog(LogLevel level, string message)
 ```
 
 ### Upgrade from 0.3.0-rc to 0.4.0-rc (For projects created by Visual Studio 2022 17.1 Preview toolkit)
+
 If there is an existing project created in VS2022 17.1 Preview, you can use the following steps to upgrade:
+
 - In `appsettings.{Environment}.json` file:
+
 1. Add `OAuthAuthority` under `TeamsFx:Authentication` and copy the value from `OAUTH_AUTHORITY`.
 2. Remove the line `"SimpleAuthEndpoint": "https://localhost:port/"`.
 3. Remove lines of configuration starting with "CLIENT_ID", "IDENTIFIER_URI", "TAB_APP_ENDPOINT", "OAUTH_AUTHORITY", "AAD_METADATA_ADDRESS", "ALLOWED_APP_IDS".
 4. Remove the Nuget dependency package "Microsoft.TeamsFx.SimpleAuth".
+
 - In Solution Explorer:
+
 1. Right click project file and choose "Manage User Secrets".
 2. Change key name "CLIENT_SECRET" to "TeamsFx:Authentication:ClientSecret".
 
+### Upgrade from 1.1.0 to 1.2.0 (Update projects to use TeamsJS V.2.0)
+
+Teams Toolkit provides users with a template `TeamsJSBlazorInterop.js`, which consists of multiple commonly used Teams JS SDK API. Users can add more APIs if needed. As suggested, even if you intend your app to only run in Teams (and not Office and Outlook), best practice is to start referencing the latest TeamsJS (_v.2.0_ or later) as soon as convenient, in order to benefit from the latest improvements, new features, and support (even for Teams-only apps).
+
+Starting from TeamsFx .NET SDK 1.3.0, TeamsJS V2 are referenced. Though previously scaffolded projects still work with TeamsFx .NET SDK 1.3.0, we suggest you replace the `TeamsJSBlazorInterop.js` under `./wwwroot/js` with the latest one from [here](https://github.com/OfficeDev/TeamsFx/blob/main/templates/tab/csharp/default/wwwroot/js/TeamsJsBlazorInterop.js). Two APIs in the file are updated with new function name to align with renamed TeamsJS APIs. `initializeWithContext()` is removed.
```

Comment:
If we are going to release in 1.2.0, this need to be changed.

Reply:
Thanks! I've changed 1.3.0 -> 1.2.0.